### ISSUE TITLE
Design issues

### ISSUE THREAD

**Issue by frokeneriksen** [OpenDesign-SUPSI/GroupA-1415#4]
I have started looking at the design - and am already discovering some limitations and suggesting some changes.
1. Maybe we should go for a toolbox on wheels - having a "thing" that we have to carry around seems a bit unnecessary and would probably be easier with something that we can "roll around" the desks.
2. The shelfs will be of the "simple" kind - since we don't have the "metal sliders". 
3. I am also thinking about combing a "tool wall" where you can hang stuff - with the toolbox, by making holes on the sides where you can hang boxes or handles. (maybe one of you would like to start designing the boxes/handles?)

**Comment by giuliarouge** [comment]
I agree with both 1 and 2, let's make it simple with embedded stuff!
Going also to help you for 3

**Comment by frokeneriksen** [comment]
I had a look at the changes you did on the new version..I don't really understand why you put in the pieces - that makes the opening smaller...and more difficult to reach whats inside..:)

**Comment by giuliarouge** [comment]
I was trying to create a structure, then we can adjust everything and cut some part, of course!

**Comment by frokeneriksen** [comment]
Ok,

I have kept you structure and added shelves and "drawers" for the outside. Plus 2 areas on the top for any tools to "not fall off" or just be able to store stuff :)

**Comment by giuliarouge** [comment]
Perfect, I am going to check and add some stuff...each of us can experiment on a piece of the structure and at the end we can choose the parts that we prefer.

**Comment by frokeneriksen** [comment]
The boxes looks fine with me:) I guess we should have more in different sizes. But that is not a big job..

**Comment by frokeneriksen** [comment]
Hi Giulia, 

I am still not sure that I understand the 4 structures you put in..I have deleted the smallest holes on the one side for.

**Comment by giuliarouge** [comment]
Yes the smallest holes were from the last model...sorry I have forgotten to delete them!
